### DISCUSSION TITLE
Failed to build nylo_framework:main:

### DISCUSSION THREAD

**Discussion by owenwattimena** [nylo-core/nylo#60] [Q&A]
i got this error when i try to run this command to make view.
"dart run nylo_framework:main make:page login_page"

Error:
Failed to build nylo_framework:main:
/C:/Users/Wentox%20Wtt/AppData/Local/Pub/Cache/hosted/pub.dev/watcher-1.0.2/lib/
src/constructable_file_system_event.dart:7:57: Error: The class 'FileSystemEvent
' can't be extended, implemented, or mixed in outside of its library because it'
s a sealed class.
abstract class _ConstructableFileSystemEvent implements FileSystemEvent {

can anyone help me to fix this please?

**Comment by agordn52** [comment]
Hi @owenwattimena,

I haven't seen this before either, what version of Flutter are you on?
Could you run `flutter doctor -v`
